internal/utils/utils.go

```diff
@@ -106,3 +106,12 @@ func (w *WaitObserver) Retry(err error) error {
 
 	return nil
 }
+
+func Contains(l []string, v string) bool {
+	for _, l := range l {
```

Comment:
Could we avoid variable shadowing please?

Reply:
Sure, nice catch! :+1: